packages/itwin/tree-widget/src/components/trees/stateless/models-tree/ModelsTree.tsx

```diff
@@ -49,18 +55,28 @@ export function StatelessModelsTree({
   filter,
   density,
   hierarchyLevelConfig,
+  hierarchyConfig,
   selectionMode,
   onPerformanceMeasured,
   onFeatureUsed,
 }: StatelessModelsTreeProps) {
-  const { getModelsTreeIdsCache, visibilityHandlerFactory } = useCachedVisibility(activeView);
+  const hierarchyConfiguration = useMemo<ModelsTreeHierarchyConfiguration>(
+    () => ({
+      elementClassGrouping: "enable",
+      elementClassSpecification: "BisCore.GeometricElement3d",
+      showEmptyModels: false,
```

Comment:
Maybe defaults should be defined as a `const` next to `HierarchyConfiguration` in `ModelsTreeDefinition.ts`? That way they could be reused here and in tests.

Reply:
Created defaults object and used that in tests.